homeassistant/components/heos/services.yaml

```diff
@@ -10,3 +10,20 @@ sign_in:
 
 sign_out:
   description: Sign the controller out of the HEOS account.
+
+join:
+  description: Group players together as a HEOS group.
+  fields:
+    leader:
```

Comment:
The entity doesn't represent the leader separately than the members, so why does the service call?  These should work the same way, otherwise it will confuse people of when to include the leader as the first element or when not to.

Reply:
Yeah, I think this is obsolete now with the new `media_player.join` service.